### DISCUSSION TITLE
Why not use alternative clients ?

### DISCUSSION THREAD

**Discussion by gnaghi** [moraroy/NonSteamLaunchers-On-Steam-Deck#528] [Ideas]
There are at least two open source alternative for Epic and GoG : Legendary and Comet (https://github.com/derrod/legendary https://github.com/imLinguin/comet). Why not use them instead of official clients ?

**Comment by moraroy** [comment]
@gnaghi I'd rather use the launchers, that way I know for sure  i will not need to use legendary nor comet to get my cloud saves and achievements as well as my online play for epic, why use more program when one program do trick

**Reply by gnaghi** [reply]
I think it would allow faster loading of games. Btw, both clients support cloud saves, but Legendary doesn't support achievements. 
So there would be two mode : compatibility, using official client, and faster, with alternative clients.

**Reply by moraroy** [reply]
@gnaghi if you uncheck no overlay in the gog settings as well as change the settings to "minimize to tray" you won't it diminishes some delay, it's about the same time delay as playing epic games tbh. I personally don't mind it. I think the big thing to fix as far as nonsteamlaunchers goes, is actually figuring out how to exit with minimize to tray. That's the key. Basically force quitting when no game is ran.

**Reply by moraroy** [reply]
@gnaghi I found a way for better load times but will need to test
